### PR TITLE
Astyle player.cpp (affected functions in commit messages)

### PR DESCRIPTION
**Pull request by valuial**
Not all of player.cpp, but some blocks of code (less danger of conflicting PRs).
Using function boundaries for boundaries of astyled blocks.

astyled:
Line 1 - Line 1800

todo:
Line 1800 - Line 13883

### CODE REVIEW

**Comment by valuial** [comment]
Just tell me if the blocks I change at once can be larger or are too big.

File: src/player.cpp [hunk 1]
```diff
@@ -850,11 +850,12 @@ void player::update_bodytemp()
                                     temp_cur[i] );
         // Produces a smooth curve between 30.0 and 60.0.
         double homeostasis_adjustement = 30.0 * ( 1.0 + scaled_temperature );
-        int clothing_warmth_adjustement = int(homeostasis_adjustement * warmth( body_part( i ) ));
-        int clothing_warmth_adjusted_bonus = int(homeostasis_adjustement * bonus_item_warmth( body_part( i ) ));
+        int clothing_warmth_adjustement = int( homeostasis_adjustement * warmth( body_part( i ) ) );
+        int clothing_warmth_adjusted_bonus = int( homeostasis_adjustement * bonus_item_warmth( body_part(
+                i ) ) );
```
Comment (Coolthulhu) [inline src/player.cpp]: This one astyled really badly.

Comment (valuial) [inline src/player.cpp]: would a linebreak after the '*' be a better option?
Or break after the '='?
I don't think leaving it as a very long line should be the solution... (Max line length has it's reason.)

Comment (kevingranade) [inline src/player.cpp]: Astyle respects existing line breaks, so if you manually wrap it somewhere
such that each line is under the limit it won't change it.

**Pull request merged**